packages/forms/src/UIForm/fields/TextArea.js

```diff
@@ -0,0 +1,54 @@
+import React, { PropTypes } from 'react';
+import FieldTemplate from './FieldTemplate';
+
+export default function TextArea(props) {
+	const { id, isValid, errorMessage, onChange, schema, value } = props;
+	const { autoFocus, description, disabled, placeholder, readOnly, title, type } = schema;
+
+	return (
+		<FieldTemplate
+			description={description}
+			errorMessage={errorMessage}
+			id={id}
+			isValid={isValid}
+			label={title}
+			labelAfter
+		>
+			<textarea
+				id={id}
+				autoFocus={autoFocus}
+				className="form-control"
+				disabled={disabled}
+				label={title}
+				placeholder={placeholder}
+				onChange={event => onChange(event, schema, event.target.value)}
```

Comment:
It's a controlled textarea. So we need to trigger the onChange to update the value in state and inject it back as props.value.
What could be overkill is the validation on each change. But this is another subject that I need to talk about with UX?